arch/riscv/compare256_rvv.c

```diff
@@ -0,0 +1,48 @@
+/* compare256_rvv.c - RVV version of compare256
+ * Copyright (C) 2023 SiFive, Inc. All rights reserved.
+ * Contributed by Alex Chiang <alex.chiang@sifive.com>
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#if RISCV_RVV
```

Comment:
Should be `#ifdef` line in `functable.c` usage?

Reply:
Thanks, it's done.